src/ghdcbot/adapters/storage/sqlite.py

```
@@ -317,6 +322,22 @@ def set_cursor(self, source: str, cursor: datetime) -> None:
                 (source, cursor_utc.isoformat()),
             )
 
+    def record_webhook_delivery(self, delivery_id: str, event_name: str) -> bool:
+        # GitHub can retry the same delivery.
+        now = datetime.now(timezone.utc).isoformat()
```

Ruff (0.15.12)

[warning] Use `datetime.UTC` alias

Convert to `datetime.UTC` alias

(UP017)